lang/en/texts/ingredients-analysis.html

```diff
@@ -0,0 +1,35 @@
+
+<img src=”https://static.openfoodfacts.org/images/misc/ingredients-analysis-example-no-text.png”alt =”Palm oil, Vegan and Vegetarian detection on the Open Food Facts website”>
+
+<p>While we understand that many vegan and vegetarian people choose to cook from raw ingredients, which is a good idea to avoid ultra-processed foods, there are some times where we have to make a snap decision in the supermarket, on a product that's not explicitly labelled vegan or vegetarian.</p>
+
+<p>Or perhaps, you're new to veganism, and you're not always familiar with the many way animal-derived ingredients that hide in the ingredient list.</p>
+
+<p>Open Food Facts will not only show you the vegan or vegetarian status of the product, based on the analysis of the ingredient list, but it will also explain why it's not vegan or vegetarian by listing the problematic ingredients.</p>
+
+<p>Please note that this analysis is based on the ingredient list only. Sometimes, processing methods such as fish bladders filtering can make an otherwise vegan product non vegan.</p>
+
+
+<p>We hope that this feature will save you time, either on your path to adopting a flexitarian, vegetarian or vegan diet, or for other reasons.</p>
+
+<p>The egg symbolizes the vegetarian diet, which allows animal derived products like milk, eggs and honey.</p>
+
+<p>The leaf symbolizes the vegan diet which only allows ingredients from vegetal sources.</p>
+
+<p>The monkey symbolizes the palm oil status, since it has been largely impacted by palm-oil related deforestation.</p>
+
+How it works
+
+<p>The detection is based on the ingredient analyzer of Open Food Facts.</p>
+
+<p>Last summer, we overhauled it to be able to assess the degree of food processing. Over the past year, it has become more multilingual and is able to deal with even more corner cases, synonyms and subtleties in wording. Based on this work, we were able to add vegan, vegetarian diets as well as a revamped palm oil detector, something we pioneered in 2012.</p>
+
+<p>The identification of vegan, vegetarian and palm oil free products is based on the ingredient analyzer of Open Food Facts. For each product, we analyze the ingredients list to separate each individual ingredient and look for the corresponding ingredient in our ingredients taxonomy.</p>
+<p>Taxonomies are where we gather a lot of information about various things like ingredients, additives, food categories…</p>
+<p>The ingredients taxonomy is a huge multilingual database of ingredients with their possible synonyms. We are constantly expanding the ingredients taxonomy in order to be able to recognize more and more ingredients in more and more languages.</p>
+<p>The support of synonyms and hierarchy in ingredients means that the parser will be handle ingredients that are often confusing to the novice ingredient list reader.</p>
```

Comment:
```suggestion
<p>The support of synonyms and hierarchy in ingredients means that the parser can handle ingredients that are often confusing to the novice ingredient list reader.</p>
```

Not sure about the "novice ingredient list reader", that could be perceived as condescending.